src/bin/console/console.cpp

```diff
@@ -358,21 +358,33 @@ void Console::out(const std::shared_ptr<const Table>& table, const PrintFlags fl
   int size_x;
   rl_get_screen_size(&size_y, &size_x);
 
-  const bool fits_on_one_page = table->row_count() < static_cast<uint64_t>(size_y) - 1;
+  const bool fits_on_one_page_y = table->row_count() < static_cast<uint64_t>(size_y) - 1;
```

Comment:
This does not take all the additional information into account:

```
(debug)> print test
=== Columns
|       a||        MVCC        |
|     int||_BEGIN|_END  |_TID  |
|not null||      |      |      |
=== Chunk 0 ===
|<ValueS>||
|       1||     0|      |      |
|      24||     0|      |      |
```

Feel free to move this into its own issue, as this has been broken before.